src/style.css

```diff
@@ -0,0 +1,268 @@
+* {
```

Comment:
Please follow the Dry, KISS AND YAGNI rule by minimizing your repeated code.

Reply:
thanks @javed